js-code-style.md

```diff
@@ -1,9 +1,20 @@
 # Javascript Code Style
 
-## eslint with the `eslint-config-airbnb-base` preset, `eslint-config-airbnb` for React projects;
+## ESLint with the `eslint-config-airbnb-base` preset, `eslint-config-airbnb` for React projects;
 
 We can start a project to extend one of those bases if we find we have issues with some rules.
 
+### For tests
+
+You should fine-tune the config for your tests folder.
+First set the right environment in your eslint config (`mocha` for example).
+There's also a set of rules that you should probably ignore/change. Consider the following:
+
+  * `no-unused-expressions`: Should be `off` when using assertion libs like `chai` for example.
+  * `prefer-arrow-callback`: Should be `off` when using test frameworks like mocha as stated [in their documentation](https://mochajs.org/#arrow-functions).
```

Comment:
If this illustrates that mocha is not a good choice of a test runner even better, I hope we can set a standard test runner that everyone agrees will play well with linters and other tools.